platform-sdk/swirlds-crypto-tss/src/main/java/com/swirlds/crypto/signaturescheme/api/PairingSignature.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.crypto.signaturescheme.api;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ *  A Mock PairingSignature
+ */
+public record PairingSignature() {
```

Comment:
Should this be called a `MockPairingSignature`?   When we search for classes, it would be useful to know immediately that the class is Mock and not a real class

Reply:
So we are all on the same page: I just wanted to let you know that nothing here is meant to be used immediately. As the task states, this is only the public interface; we defined this to help the dependent project move along and be able to see the code they depend on.
I agree that calling this mock was a problem. I have fixed that, but the intention is to have the skeleton of what will become the actual class that library users will have to invoke.
I can either remove this from the code until the actual implementation. Up to you @edward-swirldslabs, and what you think is best

Reply:
@hendrikebbers do you have a suggestion?